codesign/certificates.go

```diff
@@ -32,6 +34,8 @@ var (
 	macOSInstallerCertificateNames = []string{
 		"3rd Party Mac Developer Installer", // type: "Mac Installer Distribution"
 		"Developer ID Installer",            // type: "Developer ID Installer"
+		"Apple Development",   // type: "iOS Development for Xcode 11", id: "83Q87W3TGH"
```

Comment:
it seems `Apple Development` and `Apple Distribution` should be moved to the `macOSCertificateNames`, they are macOS codesigning certificates.

you can validate it, just create a new macOS project using Xcode 11, with default options. The project's Code Signing Identity will be set to: `Apple Development`. Now if you execute codeswigndoc against this project, it will fail: `failed to analyze archive, error: installed certificate not found with common name or sha1 hash: Apple Development: XYZ`